src/ignitus-Dashboard/ignitus-Dashboard-Header/Components/profileDropdown.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import '../Styles/style.scss';
+
+
+class ProfileDropdown extends React.Component {
+constructor(props){
+ super(props);
+
+ this.state = {
+       displayMenu: false,
+       User: this.props.User,
+     };
+
+  this.showDropdownMenu = this.showDropdownMenu.bind(this);
+  this.hideDropdownMenu = this.hideDropdownMenu.bind(this);
+
+};
+
+showDropdownMenu(event) {
+    event.preventDefault();
+    this.setState({ displayMenu: true }, () => {
+    document.addEventListener('click', this.hideDropdownMenu);
```

Comment:
Please don't use `eventListeners` in the app, we hardly make use of them in react apps theses days they are very bad  `performance` at all,  if you want to have an `hide/show` behaviour of dropdown you can do it with a single variable, I am listing `examples` that will help you understand what I meant, 

https://reactjs.org/docs/handling-events.html
https://blog.logrocket.com/building-a-custom-dropdown-menu-component-for-react-e94f02ced4a1/

Here `sidebar` is written using single variable isExpanded, that toggles the state to hide/show, https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-UserInterfaceKit/InterfaceSideNavigation/Components/interfaceSideNavigation.tsx#L31 

Here is the live version of this component. http://www.ignitus.org/interface